articles/getting-started-with-selenium-automation-testing/index.md

```diff
@@ -0,0 +1,206 @@
+
+![Step 9](/engineering-education/getting-started-with-selenium-automation-testing/step-9.jpg)
+
+Step 10: Check whether the following two library folders appear on your Eclipse dashboard. If this is the case, then you have successfully configured Selenium WebDriver with Eclipse.
+
+![Step 10](/engineering-education/getting-started-with-selenium-automation-testing/step-10.jpg)
+
+### Running The First Test in Selenium WebDriver
+
+Let’s test whether a specific URL such as `google.com` functions well. We will conduct this test using the Chrome browser. Every browser used in Selenium testing has a unique browser driver. In our case, we need to download the Chrome driver. This can be downloaded from [here]( https://sites.google.com/a/chromium.org/chromedriver/). Select the Chrome driver that corresponds to your operating system.
+
+![Chrome Driver](/engineering-education/getting-started-with-selenium-automation-testing/chrome-driver.png)
+
+After downloading the chrome driver, you need to create a class file and add import statements for necessary classes like WebDriver and ChromeDriver from Selenium. 
+
+```Java
+import org.openqa.selenium.By;  
+import org.openqa.selenium.WebDriver;  
+import org.openqa.selenium.chrome.ChromeDriver; 
+```
+
+You should write the test within the the main function.
+
+```Java
+public class FirstTest {
+  public static void main(String[] args) {
+    // Test code goes here
+  }
+}   
+```
+
+The next step involves setting the path of the downloaded executable file. The following is an example of how the property can be set.
+
+```Java
+System.setProperty("webdriver.chrome.driver", "D:\\ChromeDriver\\chromedriver.exe");
+```
+
+The following snippet is used to instantiate the driver. 
+
+```Java
+WebDriver driver = new ChromeDriver();
+```
+
+Launching the Google website requires setting its correct URL. In our case, the URL for Google is `www.google.com`. The corresponding snippet will be as follows.
+
+```Java
+driver.get("http://www.google.com/"); 
+```
+
+Let’s assume that we want to search for something on the website. The following snippet will help us to run the search. 
+
+```Java
+driver.findElement(By.name("q")).sendKeys("cheese" + Keys.ENTER); 
+```
+
+The following is the full code snippet for running the first test in Selenium WebDriver. Each code block consists of embedded comments to provide explanation. 
+
+```Java
+import org.openqa.selenium.By;  
+import org.openqa.selenium.WebDriver;  
+import org.openqa.selenium.chrome.ChromeDriver;  
+
+public class FirstTest {
+  public static void main(String[] args) {
+
+    //property setting
+    System.setProperty("webdriver.chrome.driver", "D:\\ChromeDriver\\chromedriver.exe");
+
+    //instantiating the driver
+    WebDriver driver = new ChromeDriver();
+
+    //launching the website
+    driver.get("http://www.google.com/"); 
+
+    //running a search
+    driver.findElement(By.name("q")).sendKeys("cheese" + Keys.ENTER); 

```

Comment:
@mbaabuones You missed adding the import statement for `Keys`.

Reply:
@zolomohan I have added the import statement for `Keys`